src/components/CollapsableHeader.js

```diff
@@ -0,0 +1,40 @@
+import React,{ useState } from 'react';
+import PropTypes from 'prop-types';
+import styled from 'styled-components'
+
+
+const CollapsableHeaderBody = styled.div`
+`;
+
+const CollapsableHeaderTitle = styled.div`
+    border-style: solid;
+    border-width: 1px;
+`;
+
+
+const CollapsableHeader = (props) => {
```

Comment:
You can be destructing your props if you want to:
```
const CollapsableHeader = ({ children, title }) => { ... }
```
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Destructuring_assignment